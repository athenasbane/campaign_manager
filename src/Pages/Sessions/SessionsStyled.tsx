import styled from "@mui/material/styles/styled";

import Tab from "@mui/material/Tab";

export const StyledTab = styled(Tab)(({ theme: _ }) => ({
  width: "33%",
}));
