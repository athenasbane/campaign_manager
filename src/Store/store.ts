import { combineReducers, configureStore } from "@reduxjs/toolkit";
import modalReducer from "./slices/modals";
import { contentfulApi } from "./slices/backend";
import activeMissionReducer from "./slices/activeMission";
import layoutReducer from "./slices/layout";

const rootReducer = combineReducers({
  activeMission: activeMissionReducer,
  modals: modalReducer,
  layout: layoutReducer,
  [contentfulApi.reducerPath]: contentfulApi.reducer,
});

export function setupStore(preloadedState?: Partial<RootState>) {
  return configureStore({
    reducer: rootReducer,
    middleware: (getDefaultMiddleware) =>
      getDefaultMiddleware({ serializableCheck: false }).concat(
        contentfulApi.middleware
      ),
    preloadedState,
  });
}
const store = setupStore();

// Infer the `RootState` and `AppDispatch` types from the store itself
export type RootState = ReturnType<typeof rootReducer>;

export type AppStore = ReturnType<typeof setupStore>;
// Inferred type: {posts: PostsState, comments: CommentsState, users: UsersState}
export type AppDispatch = typeof store.dispatch;

export default store;
