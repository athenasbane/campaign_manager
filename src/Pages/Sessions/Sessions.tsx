import Typography from "@mui/material/Typography";
import Session from "Components/Molecule/Session/Session";
import { useAppSelector } from "hooks/store.hooks";
import { selectSessions } from "Store/slices/sessions";
import theme from "theme";
import { SessionType } from "Types/Enum/sessions.enum";

export default function Sessions() {
  const sessionsData = useAppSelector((state) =>
    selectSessions(state.sessions)
  );

  const sessions = sessionsData.map((session) => {
    switch (session.type) {
      case SessionType.Session:
        return <Session key={session.shortDescription} session={session} />;
      case SessionType.StoryIncrement:
        return (
          <Typography
            key={session.displayLabel}
            align={session.increment === "act" ? "center" : "left"}
            variant={session.increment === "act" ? "h3" : "h4"}
            color={theme.palette.secondary.light}
            sx={{ mt: 4 }}
          >
            {session.displayLabel}
          </Typography>
        );

      case SessionType.Location:
        return (
          <Typography key={session.displayLabel} sx={{ mt: 2 }} variant="h5">
            {session.displayLabel}
          </Typography>
        );

      default:
        return <Session key={"issue"} session={session} />;
    }
  });

  return (
    <>
      <Typography align="center" variant="h3">
        Session Recap
      </Typography>
      {sessions}
    </>
  );
}
