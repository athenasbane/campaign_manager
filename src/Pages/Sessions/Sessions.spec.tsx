import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Sessions from "Pages/Sessions/Sessions";
import { Provider } from "react-redux";
import store from "Store/store";

describe("Page - Sessions", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <Provider store={store}>
        <Sessions />
      </Provider>,
      { wrapper: BrowserRouter }
    );

    expect(container).toMatchSnapshot();
  });
});
