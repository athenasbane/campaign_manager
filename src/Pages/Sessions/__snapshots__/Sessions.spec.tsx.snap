// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page - Sessions should match snapshot 1`] = `
<div>
  <h3
    class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter css-47suzj-MuiTypography-root"
  >
    Session Recap
  </h3>
  <h3
    class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter css-1fcas5g-MuiTypography-root"
  >
    Act I
  </h3>
  <h4
    class="MuiTypography-root MuiTypography-h4 MuiTypography-alignLeft css-1m0yndc-MuiTypography-root"
  >
    Chapter 2
  </h4>
  <h5
    class="MuiTypography-root MuiTypography-h5 css-1657z6w-MuiTypography-root"
  >
    Klilcaithness, The Nals
  </h5>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-h4osnm-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-controls="panel1a-content"
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
      id="panel1a-header"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Session 7 - Shin... who now?
        </p>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            class="MuiAccordion-region"
            id="panel1a-content"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <ul>
                

                <li>
                  Left the Fighting pits and headed back to the Hunters Lodge.
                  

                  <ul>
                    

                    <li>
                      Balfor had a drink with Cassius and learnt:
                      

                      <ul>
                        

                        <li>
                          Cassius was once the care taker of the fields of heroes. A changling barial ground in the Nirwod.
                        </li>
                        

                        <li>
                          His people were used as spies by the imperium of all kinds, Merchant lords looking for insider knowledge of their rivals activities. Jealous artisans looking to know the secrets to their competitions new process.
                        </li>
                        

                        <li>
                          I say used as there was very little choice involved. We are not protected by the Pact Mortalis… Therefore we are outlaws by the very definition. You could strike me down where I stand the guard would probably charge you with making a mess.
                        </li>
                        

                        <li>
                          When one of our own died I was responsible for their burial.
                        </li>
                        

                        <li>
                          A few years ago the however a not a body but a pair dressed in black and with silver masks came to our lands.
                        </li>
                        

                        <li>
                          Maybe the inquisition thought we worshiped our ansestors who knows they never said but they raised the place to the ground and made me watch
                        </li>
                        

                        <li>
                          They branded me so my changes would never be convincing again.
                        </li>
                        

                        <li>
                          Then the real fun began. The put me in a chamber and I was bathed in burning light.
                        </li>
                        

                        <li>
                          a year this went on for.
                        </li>
                        

                        <li>
                          in a fit of anger, spite and fear prayed to Hirane thinking is this what you want to see a creature burn in your name?
                        </li>
                        

                        <li>
                          I didn’t hear an answer but the next time that machine was turned on the light didn’t burn it healed. The Inquisitors in charge of my tourture started to look for me even though I was right in front of them.
                        </li>
                        

                        <li>
                          Since then their “eyes” have passed right over me. I took the holy symbol of Hirane to this day,
                        </li>
                        

                        <li>
                          Interesting isn’t it. That me anointed by the heart of the pantheon can’t be seen by them… why is it that their professed god would hide me so?
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                    <li>
                      Found out the thieves of the bakeries were a group of Kenku that called themselves “The Snagpies” that were just staving. Gormar paid them 10 gold each to just go away.
                    </li>
                    

                    <li>
                      Karlyle slept with Wendy the barmaid who was mysteriously then found dead by poison the next morning. The group suspected Dick at first but then followed the part of Karlyles cloak she had using a spell to Hacker’s Herbalism. They fought and killed her and found out she was a member of the Shin Duku. A mysterious assassins guild.
                    </li>
                    

                    <li>
                      Balfor was sent a magical bird by his master Ofridi that told him to meet him by a tree. Ofridi appeared out of the tree then gave Balfor a sending stone and told him to continue on his current path.
                    </li>
                    

                  </ul>
                  

                </li>
                

              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-h4osnm-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-controls="panel1a-content"
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
      id="panel1a-header"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Session 6 - Killing in the Name of... Krig
        </p>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            class="MuiAccordion-region"
            id="panel1a-content"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <ul>
                

                <li>
                  Completed the Nessy Brewery mission and levelled up.
                  

                  <ul>
                    

                    <li>
                      Sold and bought a load of stuff including the brazer
                    </li>
                    

                    <li>
                      Went to the university and learnt a lot on various subjects
                    </li>
                    

                    <li>
                      Spoke to Heik the feather is ready but you can leave when you want.
                    </li>
                    

                    <li>
                      Fought in the Fighting Pits of Krig managed to get to the get to the 5th Round and tied with the Fate’s Fortune who you met there.
                    </li>
                    

                  </ul>
                  

                </li>
                

              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-h4osnm-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-controls="panel1a-content"
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
      id="panel1a-header"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Session 5 - Action in the Ashglade
        </p>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            class="MuiAccordion-region"
            id="panel1a-content"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <ul>
                

                <li>
                  Killed the remaining vampire spawn at the ashglade and pearson farms.
                  

                  <ul>
                    

                    <li>
                      Was told that Ulric will release the rest of his Bailey stores to the ness brewery.
                    </li>
                    

                    <li>
                      Went north met with some gobs killed them.
                    </li>
                    

                    <li>
                      Then was tricked by a cowboy dwarf
                    </li>
                    

                    <li>
                      It was lamia that was revealed and attempted to call a Manticore to him.
                    </li>
                    

                    <li>
                      You killed the lamia then the manticore and raided the Lamia’s lair that was a strange scaled up hall.
                    </li>
                    

                    <li>
                      You found an injured real Gizmug in the lair. He told you that it’s “The Collector” that wants the magical beasts. He used a scroll to cross off the Manticore and an elf appeared he “introduced” himself as Elromior he paid you for both the Manticore and the Lamia.
                    </li>
                    

                  </ul>
                  

                </li>
                

              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-h4osnm-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-controls="panel1a-content"
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
      id="panel1a-header"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Session 4 - Welcome to Klilcaithness
        </p>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            class="MuiAccordion-region"
            id="panel1a-content"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <ul>
                

                <li>
                  Spoke to Hidii Grunbumn about franzi she gave you a reward for letting her know.
                  

                  <ul>
                    

                    <li>
                      

                      <p>
                        went aboard the feather still damaged from the dinosaur attack to klilcaithness
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        Got to klilcaith bought somethings at the Griffin enchanting services
                      </p>
                      

                      <ul>
                        

                        <li>
                          Was told that they sell less than legal magic items out the back of the shop by someone who speaks thieves cant Druggen
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        Went to the brewery and was told that they had to increase the price of their ale due to their barley supplier stopping growing barley and the owners of the farm going missing
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        Manticore and 5gp per set of ears of gobilins
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        They “bought” some horses, a cart and headed off you where attacked by knolls then went to the farm house.
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        First you saw a boy that's under the influence of quinine working and carrying produce of a strange flowering plant
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        then you met a young woman who due to a natural 20 perception check from Karlyle he noticed this young woman wasn't what she seemed.
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        You defeated them and found a note.
                      </p>
                      

                      <p>
                        Production is lower than expected make sure you make up the difference.
                      </p>
                      

                      <p>
                        Lamuin
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        you also noticed there was 4 glasses of blood.
                      </p>
                      

                    </li>
                    

                  </ul>
                  

                </li>
                

              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <h4
    class="MuiTypography-root MuiTypography-h4 MuiTypography-alignLeft css-1m0yndc-MuiTypography-root"
  >
    Chapter 1
  </h4>
  <h5
    class="MuiTypography-root MuiTypography-h5 css-1657z6w-MuiTypography-root"
  >
    The Shambles, The Nals
  </h5>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-h4osnm-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-controls="panel1a-content"
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
      id="panel1a-header"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Session 3 - Inker Investigation
        </p>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            class="MuiAccordion-region"
            id="panel1a-content"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <ul>
                

                <li>
                  

                  <p>
                    came from the wastes
                  </p>
                  

                  <ul>
                    

                    <li>
                      Attacked by a large monster The Feather damaged
                    </li>
                    

                    <li>
                      Went back to The Shambles spoke to Thorsten went on a mission to get Georg Inkers ledger, was successful, got the reward.
                    </li>
                    

                    <li>
                      Went back to Vandrad she give you a new mission
                      

                      <ul>
                        

                        <li>
                          You each have a promption in imperial alignment.
                        </li>
                        

                        <li>
                          Also let the mother of Franzi know that you have information out her.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                  </ul>
                  

                  <p>
                    <strong>
                      Vandrads mission:
                    </strong>
                  </p>
                  

                  <ul>
                    

                    <li>
                      

                      <p>
                        As you no doubt have gathered by now I have a personal interest in lost things… A young and promising enchanter by the name of Bolli Purlic.
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        After studying hard here in Old Shamphel he was offered a very good job for a master artisian in Thorpness.
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        The boy seems head strong, managed to avoid any of the nasty drug business the youth of rich seem to be interested in these days and was making a name for himself as a rising star in the enchanting circles.
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        He left that job a few months ago and has not been heard from since. This sudden leaving seems to have coincided with the rise of an Unaligned Group that call themselves “The Raven” that have been conducting raids on Taxation vessels coming from the north
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        We believe that Purlic has some how got mixed up in this. We intercepted an agent of The Raven, here in Shamphel, a nasty little man by the name of Ceolmon. He had a note from Purlic:
                      </p>
                      

                      <p>
                        Ceolmon,
                      </p>
                      

                      <p>
                        It seems our usual ways of communication are being intercepted some how so this is a matter of last resort. Please send aid in the form of people we can trust to Thorpness. This is a tricky situation so please take your time in picking the right ones.
                      </p>
                      

                      <p>
                        I'll be in the usual place every morning first light doing my usual activity.
                      </p>
                      

                      <p>
                        <em>
                          Bolli
                        </em>
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        From what we know Purlic is enjoys to paint we suspect this is the “Usual Activity” that he describes.
                      </p>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        The usual place from some of his previous works:
                      </p>
                      

                      <ul>
                        

                        <li>
                          Shows very well painted scenes all from above of people in motion.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                    <li>
                      

                      <p>
                        It seems that he has a particular subject he prefers.
                      </p>
                      

                    </li>
                    

                  </ul>
                  

                  <p>
                    <strong>
                      Your mission:
                    </strong>
                  </p>
                  

                  <ul>
                    

                    <li>
                      Go undercover as the support requested from Purlic. Work with him to understand his involvement with The Raven. Gain an understanding of their organisation.
                    </li>
                    

                    <li>
                      I cannot stress enough that no harm must come to Purlic… He is to be protected.
                    </li>
                    

                    <li>
                      However, once you have had enough time to assess the situation you should try to convince him to return to us here. If you can do that of his own accord that would be preferable. However, if this cannot be done other than force please return with the information only.
                    </li>
                    

                  </ul>
                  

                  <p>
                    Reward:
                  </p>
                  

                  <ul>
                    

                    <li>
                      For successful gaining of information I will award you:
                      

                      <ul>
                        

                        <li>
                          100… PC… each…
                        </li>
                        

                        <li>
                          If you convince him to return here a further 50PC will be granted.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                  </ul>
                  

                  <p>
                    Logistics:
                  </p>
                  

                  <ul>
                    

                    <li>
                      You will meet the Southver boat crew at the Southver flatts they will go to Kilcathness for some necessary repairs they need to do to their ship. Once complete you will continue on to Thorpeness and your mission.
                    </li>
                    

                  </ul>
                  

                </li>
                

              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-h4osnm-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-controls="panel1a-content"
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
      id="panel1a-header"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Session 2 - Wading in the Wastes
        </p>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            class="MuiAccordion-region"
            id="panel1a-content"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <ul>
                

                <li>
                  travelled into the wastes on board the feather
                  

                  <ul>
                    

                    <li>
                      Were attacked by goblins and hobgoblins
                      

                      <ul>
                        

                        <li>
                          Both Balfor and karlyle downed but Quick thinking from Dick and Balfor respectively saved their lives.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                    <li>
                      Met a Lizardfolk monk that spoke to Dick let him know that Kelzis is likely to the North In Thorpeness where his father ‘The Amassador’ was last.
                      

                      <ul>
                        

                        <li>
                          She directed you too the new village to find Samual. There you engaged with the villagers that seemed slightly changed.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                    <li>
                      After killing them, or “helping them” for Dick, you engaged with a mutated lizardfolk sharman.
                    </li>
                    

                    <li>
                      in his lair you found
                      

                      <ul>
                        

                        <li>
                          an old map that listed the current location as “The Hall of Hirane and Avanker” the formal name for the Inquisition
                        </li>
                        

                        <li>
                          A spell book written in both Dragonic and Abysal that was some sort of instructions for summoning.
                        </li>
                        

                        <li>
                          A section of a personal journal that had two things repeated:
                          

                          <ul>
                            

                            <li>
                              I will be come one with them, them with me, my people will come, all will be one.
                            </li>
                            

                            <li>
                              Pergash bound, the path is laid. The take the stones from the glade.
                            </li>
                            

                          </ul>
                          

                        </li>
                        

                      </ul>
                      

                    </li>
                    

                    <li>
                      a young boy in a cage called Samuel
                      

                      <ul>
                        

                        <li>
                          A PseudoDragon that Balfour took as a familiar.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                  </ul>
                  

                </li>
                

              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-h4osnm-MuiPaper-root-MuiAccordion-root"
  >
    <div
      aria-controls="panel1a-content"
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
      id="panel1a-header"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Session 1 - The Journey Begins
        </p>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
        >
          <div
            aria-labelledby="panel1a-header"
            class="MuiAccordion-region"
            id="panel1a-content"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
            >
              <ul>
                

                <li>
                  Joined together
                  

                  <ul>
                    

                    <li>
                      Completed the Clearing of Shelmark
                      

                      <ul>
                        

                        <li>
                          The Party found the body of Franzi they were going to give it to Luka but Balfor intuited that it maybe worth something to someone. Gormar's sword seemed attracted to her then when it touched the head with the pomel a helm appeared Gormar put it on and it disappeared.
                        </li>
                        

                        <li>
                          The party still have her armour.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                    <li>
                      The gang stayed in a in the The Half-full Chalice. Karlyle hooked up with Metha but due a poor performance she left before he woke.
                    </li>
                    

                    <li>
                      The gang then met with Rosalina
                      

                      <ul>
                        

                        <li>
                          she's told then that the transport to the wastes will be at Southver Flats in the morning.
                        </li>
                        

                        <li>
                          Dick stole 5CP of the Reward money.
                        </li>
                        

                        <li>
                          She mentioned her husband died by goblins recently in Hiranemount.
                        </li>
                        

                      </ul>
                      

                    </li>
                    

                  </ul>
                  

                </li>
                

              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
